src/Tizen.NUI/src/public/BaseComponents/LottieAnimationView.cs

```diff
@@ -1502,7 +1502,7 @@ static internal void RootCallback(int id, int returnType, uint frameNumber, ref
 
             if (weakReferencesOfLottie.TryGetValue(id, out current))
             {
-                if (current.TryGetTarget(out currentView) && (currentView != null) && !currentView.IsDisposedOrQueued)
+                if (current.TryGetTarget(out currentView) && (currentView != null) && !currentView.IsDisposedOrQueued && current != null)
```

Comment:
이미 앞에서 current에 접근하고 있어서
```suggestion
                if (current != null && current.TryGetTarget(out currentView) && (currentView != null) && !currentView.IsDisposedOrQueued)
```
로 바꾸는게 좋지 않을까요?

Reply:
리뷰 감사합니다! 
네. 맞습니다. 제 실수 입니다. ^^;